tests/functional/object/mcg/test_bucket_policy.py

```diff
@@ -1286,3 +1291,65 @@ def test_supported_bucket_policy_operations(self, mcg_obj, bucket_factory):
         assert (
             not missing_policies
         ), f"Some bucket_policies are not created : {missing_policies}"
+
+
```

Comment:
add the customer bug jira id

Reply:
@PrasadDesala Recently all the bugzilla ids were removed from all the test cases. On my discussion with Elad, he said there is no need to add bugzilla or jira ids if the bug is closed with fix.

Reply:
We have been following the practice of including GSS bug IDs in the test for tracking purposes. Due to the transition from Bugzilla to Jira, I think we may need to revisit this approach and decide on the best way to capture this bug metadata. Perhaps mentioning the bug ID in the test case docstring could be one way to do this but there may also be other approaches we can consider, which we need to discuss and decide on

Reply:
ack. for now i have mentioned the bug id in the test case doc strings